app/controllers/works_controller.rb

```diff
@@ -20,6 +20,8 @@ def show
 
   def new
     collection = Collection.find_by!(druid: params.expect(:collection_druid))
+    @collection_title = collection.title
```

Comment:
The changes in here make me wonder if we want to add `collection_title` as an attr to the `WorkForm`, the way we're handling the `collection_druid`, and then we'd access it via the form instance instead of via a new instance variable. What's currently here is a-ok. (If you have thoughts on this, I'm all ears, but if you don't, you can also disregard!)

Reply:
I spent a little time trying this just now, since we have to create the instance variable in three different actions. However, where I got stuck was the `edit` action. It uses `ModelSync::Work.call` and the collection title is not in the work or the cocina. I wasn't sure how to get it into the form in that case. If you have an idea how to do that, let me know.

Reply:
OK, thanks for giving that a whirl. We could probably make a case for grabbing the collection title in `ModelSync::Work` but this is far enough from breadcrumbs that it may not be worth more of our time.